.github/workflows/ci.yml

```
@@ -83,36 +83,30 @@ jobs:
         name: dist
 
   publish:
-    name: Publish to Github and Pypi
+    name: Publish release to Github and Pypi
     runs-on: ubuntu-latest
     needs: [lint, test, build]
     if: success() && startsWith(github.ref, 'refs/tags/v')
     steps:
     - uses: actions/checkout@v3
-    - name: Download wheel artifact
-      uses: actions/download-artifact@v1.0.0
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
       with:
-        name: dist
-    - name: Get version from git ref
-      id: get_version
-      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
-    - name: Create GH release
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        python-version: '3.9'
+    - name: Download assets
+      uses: actions/download-artifact@v3
       with:
-        tag_name: ${{ steps.get_version.outputs.VERSION }}
-        release_name: Release ${{ steps.get_version.outputs.VERSION }}
-        draft: false
-        prerelease: false
-    - name: Upload release assets
+        name: dist
+        path: dist
+    - name: Release
       uses: softprops/action-gh-release@v1
       with:
-        tag_name: ${{ steps.get_version.outputs.VERSION }}
-        files: 'dist/*.tar.gz;dist/*.whl'
         token: ${{ secrets.GITHUB_TOKEN }}
+        files: |
+          dist/*.tar.gz
+          dist/*.whl
     - name: Publish to PyPI
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
         user: __token__
-        password: ${{ secrets.PYPI_PASSWORD }}
+        password: ${{ secrets.PYPI_PASSWORD }}
```

```suggestion
        password: ${{ secrets.PYPI_PASSWORD }}

```

Finish file with new line.